ceph.spec.in

```diff
@@ -536,6 +577,21 @@ librbd1 or libcephfs1 instead.
 
 %endif
 
+%package compat
```

Comment:
Why don't we just build/ship an empty ceph package that does exactly this? I'd say people are quite used to 'ceph' being a shortcut for these 3 daemons or meaning something like 'default install'.